airflow/api_fastapi/core_api/datamodels/assets.py

```diff
@@ -64,3 +64,45 @@ class AssetCollectionResponse(BaseModel):
 
     assets: list[AssetResponse]
     total_entries: int
+
+
+class DagRunAssetReference(BaseModel):
+    """Serializable version of the DagRunAssetReference ORM SqlAlchemyModel."""
```

Comment:
```suggestion
    """DAGRun serializer for asset responses."""
```

(We need to remove the ORM Sqlalchemy reference. This does not only serialize SQLAlchemy objects)

Reply:
Done